agot-bg-game-server/src/client/GameLogListComponent.tsx

```diff
@@ -640,14 +640,17 @@ export default class GameLogListComponent extends Component<GameLogListComponent
 
             case "retreat-region-chosen":
                 return <>
-                {data.regionTo ?
```

Comment:
The migration for old logs is missing where regionTo might be null. Maybe you want to replace such logs by your new retreat-failed log.

Reply:
Sure, if I add a migration I think I'll also move this to go into `v1.4-branch` as this isn't really a priority issue and there's a couple of migrations there already